samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -57,10 +57,28 @@ See `Fast Pair Advertising`_ for detailed information about discoverable and not
 Fast Pair device registration
 =============================
 
-Before you can use a device as a Fast Pair Provider, you must register the device model with Google.
+Before you can use your device as a Fast Pair Provider, the device model must be registered with Google.
 This is required to obtain Model ID and Anti-Spoofing Private Key.
+You can register your own device or use the debug Model ID and Anti-Spoofing Public/Private Key pair obtained by Nordic for development purposes.
+By default, if Model ID and Anti-Spoofing Private Key are not specified, the following debug Fast Pair provider is used:
+
+* NCS Fast Pair demo - The input device Fast Pair provider:
+
+   * Device name: NCS Fast Pair demo
+   * Model ID: ``0x2A410B``
+   * Anti-spoofing key (base64, uncompressed): ``Unoh+nycK/ZJ7k3dHsdcNpiP1SfOy0P/Lx5XixyYois=``
+   * Type: Input device
+   * Additional features: Data only connection, Support personalized name, Ringing device unsupported
```

Comment:
Can we use some kind of template doc mechanism for including this part of the documentation? We would need to have the possibility to provide some input arguments to it (e.g. Model ID = 0x2A410B, key, type, features)

The idea is to reuse the template in other places (e.g. nRF Desktop) and have a uniform way of phrasing this part of the documentation that is consistent across all projects.

@annwoj, @divipillai, is it possible?

Reply:
yeah, I had the same idea. We can have include.txt for this content. If this cannot be fixed now, we can fix this in a separate PR.

Reply:
Good idea, I will work on it on monday, but if I don't manage to do it before release I will do it later.